### PR TITLE
(feat) 009: add Optimisation Module agenda

### PR DESCRIPTION
**Pull request by valeriashust**

### CODE REVIEW

**Review by Dzmitry-Klokau — APPROVED** [review]

File: optimisation/index.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Intro (how does RN works, why performance issues can be faced)
+2. How to measure performance of RN app
+3. Common ways to improve performance (console statements; FlatList; React.memo(); useMemo()/useCallback(); images: sizes and format, caching; aminations; unnecessary dependencies; Reselect; inline anonymous functions
```
Comment (unfalse) [inline optimisation/index.md:5]: aminations -> animations

File: optimisation/index.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
+4. Hermes
```
Comment (iralitv) [inline optimisation/index.md:6]: What do you think about adding Analytics section here? (Crashlytics/Firebase/Sentry)?

Comment (Liza-Veis) [reply optimisation/index.md:6]: will cover this in Analytics module #58

File: optimisation/index.md (hunk 1 above)
Comment (unfalse) [inline optimisation/index.md:5]: How about adding Fast refresh? It could increase the coding performance
https://reactnative.dev/docs/fast-refresh

Comment (valeriashust) [reply optimisation/index.md:5]: I don't see how Fast Refresh is connected to app's optimisation. It's just a development tool

**Review by adenyx — APPROVED** [review]

**Pull request merged**